packages/hdwallet-native/src/__tests__/sui.test.ts

```
@@ -0,0 +1,77 @@
+  it("should support SUI", () => {
+    expect(core.supportsSui(wallet)).toBe(true);
+  });
+
+  it("should generate SUI address", async () => {
+    const address = await wallet.suiGetAddress({
+      addressNList: core.suiGetAccountPaths({ accountIdx: 0 })[0].addressNList,
+    });
+
+    expect(address).toBeTruthy();
+    expect(address).toMatch(/^0x[0-9a-f]{64}$/);
+    console.log("SUI Address:", address);
```

ESLint

[error] Unexpected console statement.

(no-console)